lib/chef/log.rb

```diff
@@ -28,8 +28,13 @@ class Chef
   class Log
     extend Mixlib::Log
 
+    def self.use_default_settings!
+      init(MonoLogger.new(STDOUT))
+      nil
+    end
+
     # Force initialization of the primary log device (@logger)
-    init(MonoLogger.new(STDOUT))
+    use_default_settings!
```

Comment:
this should probably just be `reset!` or something like that.

Reply:
This already has a `reset!` method from `Mixlib::Log`. As far as I can tell, `init(MonoLogger.new(STDOUT))` is the same as calling `init` with no arguments, so a plain `reset!` would probably work, but I didn't want to introduce any weird side effects.

Reply:
what about calling it `setup!`? (and i'm only picking a nit over the name, didn't realize that there was already a `reset!` method)